app/models/report.rb

```diff
@@ -11,7 +11,6 @@ class Report < ApplicationRecord
   has_many :transactions
   has_many :planned_disbursements
 
-  validates_uniqueness_of :fund, scope: :organisation
```

Comment:
This makes sense to remove but makes me wonder why this constraint was here to begin with. Do we know?

Reply:
I put it down as a simple mistake, there is no way a report can be unique for a fund inside the scope of an organsiation,  

I imagine it was missed becase our tests don't get setup with prior reports and we only have one report per fund and organisation right now, as soon as I approved one and tried to create the new one the error was clear - and that drove this change.